lib/stages/CMakeLists.txt

```diff
@@ -67,7 +67,9 @@ add_library(
     VisSharedMemWriter.cpp
     DataQuality.cpp
     BeamExtract.cpp
-    BeamInspect.cpp)
+    BeamInspect.cpp
+    testDataGen.cpp
```

Comment:
Is there a reason why `testDataGen` has been moved to the `stages` directory?

Reply:
Yes, because without `-DWITH_TESTS=on` the normal kotekan builds (including the site build) won't have access too it. And since it's actually very useful, I figured we should build by default.